nipype/interfaces/cat12/preprocess.py

```diff
@@ -0,0 +1,363 @@
+import os
+import traits
+from nipype.interfaces.base import InputMultiPath, TraitedSpec, isdefined
```

Comment:
We use `nipype.interfaces.base` to override some aspects of traits. I would suggest the following:

```suggestion
from nipype.interfaces.base import InputMultiPath, TraitedSpec, isdefined, traits, File, Str
```

Reply:
Hi @effigies sorry for the late reply, I only had the time to correct you reviews today. I do not known what you mean by make specs...